dbms/src/Flash/Mpp/MPPTaskManager.cpp

```diff
@@ -70,6 +70,34 @@ MPPTaskPtr MPPTaskManager::findTaskWithTimeout(const mpp::TaskMeta & meta, std::
     return it->second;
 }
 
+static Poco::Logger * getLogger()
+{
+    static Poco::Logger * logger = &Poco::Logger::get("MPPTaskCancelFunctor");
+    return logger;
```

Comment:
use `Logger::get("MPPTaskCancelFunctor", task->getId()->toString())`?

Reply:
I think we don't need to log task id here since it is not task specific, if move failed, all task's move will fail.

Reply:
ok.
replace by `Logger::get("MPPTaskCancelFunctor)`?
Poco::Logger is deprecated in TiFlash.

Reply:
OK